pkg/pgtune/parallel.go

```diff
@@ -31,13 +31,14 @@ var ParallelKeys = []string{
 
 // ParallelRecommender gives recommendations for ParallelKeys based on system resources.
 type ParallelRecommender struct {
-	cpus int
+	cpus         int
+	MaxBGWorkers int
```

Comment:
this doesn't need to be exported, it appears. I prefer to keep things unexported to reduce API "scope" until we're sure we need them

Reply:
as discussed, I can't seem to write new tests without doing this